Engine/Reflection/DotNetTypeData.cs

```diff
@@ -184,5 +184,18 @@ ITypeData getTypeDescriptor()
 
         /// <summary> Gets a string representation of this CSharpType. </summary>
         public override string ToString() => $"[{Name}]";
+
+        /// <summary> Equality for MemberData. Returns true if the other object is a MemberData and refers to same member object. </summary>
+        public override bool Equals(object obj)
+        {
+            if (obj is MemberData m && m.Member == Member)
+                return true;
+            return false;
+        }
+        /// <summary> GetHash for MemberData. </summary>
+        public override int GetHashCode()
+        {
+            return Member.GetHashCode() * 32143211;
```

Comment:
What is the point of this multiplication?

Reply:
I scramble the hash to reduce the risk of collisions. Otherwise they'd have the same hash value as the Member and this would not be every "random".

Reply:
It's not super important. If there were more things being hashed together it would be.